### PR TITLE
use internal url for idam callback

### PR DESCRIPTION
**Pull request by reddy1praveen**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: app/components/security.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ const {URLSearchParams} = require('url');
 const FormatUrl = require('app/utils/FormatUrl');
 
 const getUserToken = (hostname, applicationId) => {
-    const redirect_url = FormatUrl.format(hostname, config.services.idam.caveat_redirectUrl);
+    //ignore host name 
+    //const redirect_url = FormatUrl.format(hostname, config.services.idam.caveat_redirectUrl);
+    const redirect_url = FormatUrl.format(config.services.idam.caveat_redirect_base_url, config.services.idam.caveat_redirectUrl);
```
Comment (LucaDelBuonoHMCTS) [inline app/components/security.js]: Remove commented out lines

File: infrastructure/main.tf [hunk 2]
```diff
@@ -14,6 +14,7 @@ locals {
   //localBusinessServiceUrl = "http://probate-business-service-${var.env}.service.${local.aseName}.internal"
   //businessServiceUrl = "${var.env == "preview" ? "http://probate-business-service-aat.service.core-compute-aat.internal" : local.localClaimStoreUrl}"
   // add other services
```
Comment (LucaDelBuonoHMCTS) [inline infrastructure/main.tf]: Remove commented out lines

File: app/components/security.js (hunk 1 above)
Comment (Wivenhoe-HMCTS) [reply app/components/security.js]: removed

File: infrastructure/main.tf (hunk 2 above)
Comment (Wivenhoe-HMCTS) [reply infrastructure/main.tf]: done

**Comment by SamanthaSJ** [comment]
Verified PR Passed.

**Pull request merged**